src/components/stepper/StepperCircle.tsx

```diff
@@ -0,0 +1,28 @@
+import {MainContainer, Circle, Title, Step ,Img} from "./StepperCircle.styles"
+import CheckIcon from "../../assets/check.png"
+import Arrow from "../../assets/arrow.png"
+
+interface StepperCircleProps {
+    numberOfStep: number;
+    title: string;
+    currentStep: number;
+}
+
+function StepperCircle ({numberOfStep, title, currentStep}: StepperCircleProps){
+    
+    const isBackStep= numberOfStep< currentStep;
+    
+    const isCurrentStep=numberOfStep==currentStep
+    
+    return (
+        <MainContainer>
+            {isCurrentStep? 
+                <Img src={Arrow}></Img>:""}
+                <Circle $isBackStep={isBackStep} $isCurrentStep={isCurrentStep}>
+                    {isBackStep ? <img src={CheckIcon}></img> : numberOfStep}
+                </Circle>
+                <Title>{title}</Title>
+        </MainContainer>
```

Comment:
La flechita, es mejor si la posicionamos dentro del `<Circle />` para que al darle la posicion, lo hagas respecto al contenedor del circulo y no respecto al MainContainer


```suggestion
        <MainContainer>
            <Circle 
                $isBackStep={isBackStep} 
                $isCurrentStep={isCurrentStep}
            >
                { isCurrentStep && <Img src={Arrow} /> }
                { isBackStep ? <img src={CheckIcon}></img> : numberOfStep }
            </Circle>
            <Title>{title}</Title>
        </MainContainer>
```